### PR TITLE
Demo Finished: Requesting Feedback

### PR DESCRIPTION
**Pull request by JoarEkelund**
Hi!

Finished our Demo
Link to video: [Screencast](https://youtu.be/irbcl3ipwao)

### CODE REVIEW

**Comment by gluckzhang** [comment]
Hi @JoarEkelund , thanks for the submission. In order to have specific feedback on an early submission, you need to follow the instructions here to register your request:

https://github.com/KTH/devops-course/blob/master/contributions/feedback-planning.md

You could either do it in this PR together, or make another PR that only registers the feedback request.

**Comment by JoarEkelund** [comment]
Too late now, but still a final submission. :)

**Comment by gluckzhang** [comment]
No problem, the PR has been merged :)

**Comment by christinerosquist** [comment]
I can give you feedback on the demo! :)

**Comment by Pontus-Broberg** [comment]
I would like to feedback this demo!

**Comment by nour95** [comment]
Hi, Can I give you feedback?

**Comment by adbjo** [comment]
Hello, I would like to give feedback on this demo!

**Comment by JohanKJIP** [comment]
Good and interesting demo! 

Regarding the criteria, I would say this demo fulfils:
- The demonstration screencast is difficult to do, yes
- The demonstration screencast is sublime (eg visually appealing), yes 
- An accompanying Github repository has been made (optional), yes

However, I think the demo would really benefit from voice over since it is hard to follow what you do on the screen and read the subtitles at the same time.

**Comment by nour95** [comment]
Hi!
The demo is good and clear. 
However, it can be clearer as @JohanKJIP  wrote.
moreover, I assume that the video in minute 0:13 show an easter egg or a fun fact, but I personally didn't understand it. Can you write a subtitle or something?

**Comment by christinerosquist** [comment]
## Feedback

Great demo and interesting topic! I think that you motivate really well why the screencast is relevant in DevOps. Something that can be improved is that you don’t always explain everything that is happening in the demo, so it got a bit confusing to keep track sometimes. For example, when typing “Hudson” in the beginning. What does that mean? I found that part a bit confusing. There are a lot of time to add more subtitles to the demo in order to make it more clear what you are doing at the different steps. I would also suggest to make the subtitles easier to read, maybe add like a textbox with a background or something to show the subtitles on (the black text can be hard to read sometimes). I did not find an easter-egg in the demo so I guess you skipped that part, or did I miss something? Good job!!

**Comment by Pontus-Broberg** [comment]
Great demo! 
I like the pace of it and you give the viewer more than enough time to read all the subtitles. The subs were sometimes hard to read however so I would suggest having some background to them to give the text more contrast. 
It was great that you showed the side-by-side view at the end, where you pushed the code and showed that the webhook triggered. However, this part and the parts after that had no subtitles explaining what you were showing on the dashboards. Some explanations in the end there while you show us would be great.

Overall a great demo. Good job!

**Comment by adbjo** [comment]
Good job on the demo, I like it!
Nice animations at the beginning. The video quality is great and the pacing is easy to keep up with.

Here is some feedback:
- The pace may be a bit too slow, speed up command line typing parts and keep the full command/text for longer.
- The video don't have any subtitles at the end, this is a bit confusing since the first half has a lot of subtitles.
- What does the Hudson/Jenkins animation at 0:13 mean? Adding some text here would be great, the animations are cool.

**Comment by johansettlin** [comment]
@nour95 , @christinerosquist, @adbjo 
The easter egg is not for the uninitiated and requires some background knowledge. Here you go: https://www.devteam.space/blog/continuous-integration-hudson-vs-jenkins/

We think that an easter egg should be hidden and maybe therefore not explained in the video.
Thanks for all the feedback.

**Comment by hittesharma** [comment]
Work well done. Webhook implementation was good. Just a few comments to make it better. You can add more technical things like by demonstrating how you can customize the criteria of build failures in Sonarqube (code smells, coverage threshold etc.). It is a great tool and I have many times integrated it with Jenkins, I understand how it behaves if you don't pick a specific version of sonarqube with a particular version of Jenkins. So for that, I appreciate your work.
